### PR TITLE
time: Added the functions mktime localtime and gmtime

### PR DESCRIPTION
**Pull request by rguillon**
A simple implementation of the functions mktime localtime and gmtime.

I didn't add a test/example file, I could if you think it's needed.

### CODE REVIEW

**Comment by pfalcon** [comment]
Thanks. Sorry, I missed to respond to your forum post, and actually was even ready to ack adding it on C side. I'm glad you tried ffi though, that's how it was originally intended to be done (i.e. if it's possible to do it on Py side, do it there). One question though - did you test this on both 32-bit and 64-bit platforms?

**Comment by rguillon** [comment]
Nope I only ran my tests on a 64bits one. Would setting up a 32bit VM on a 64bits platform cover this case?

**Comment by pfalcon** [comment]
Just build uPy with "make MICROPY_FORCE_32BIT=1" (after clean). Assuming you're on x86 of course.

**Comment by rguillon** [comment]
Ok nice, it's working on both platforms. I've added a little example usable like micropython tests (it prints the same results from micropython and cpython3.5)

File: time/example_time_tuple.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import time
+
+ts=0
```
Comment (pfalcon) [inline time/example_time_tuple.py]: Sorry for backlogging with merging this. Unfortunately, there're multiple PEP8 inconsistencies in this example, so I have to request to fix them.

Comment (rguillon) [inline time/example_time_tuple.py]: Fixed, sorry about that.

**Comment by pfalcon** [comment]
Merged, thanks! Update pushed to PyPI.

**Pull request closed without merging**